src/api/component.js

```diff
@@ -189,6 +191,7 @@ Component.prototype = Object.create(HTMLElement.prototype, {
   createdCallback: {
     configurable: true,
     value() {
+      setElementAsDefined(this);
```

Comment:
I thought this needs to be called here for V0 support, otherwise it will never be called since it has been extracted from `callConstructor`? Maybe Im wrong, but tests works for me on Chrome.